apps/admin-gui/src/app/shared/components/dialogs/create-sponsored-member-dialog/create-sponsored-member-dialog.component.ts

```diff
@@ -52,6 +52,8 @@ export class CreateSponsoredMemberDialogComponent implements OnInit {
   namespaceControl: UntypedFormGroup = null;
   selectedSponsor: User = null;
   sponsorType = 'self';
+  languages = ['en'];
+  currentLanguage = 'en';
```

Comment:
If we are sure "en" will always be an option it's ok, otherwise we could assign the supported languages here and choose the first one as current. Same for the genereate sponsor.